src/main/java/org/helpberkeley/memberdata/v300/WorkflowBeanV300.java

```diff
@@ -69,7 +69,7 @@ public class WorkflowBeanV300 implements WorkflowBean {
     public WorkflowBeanV300() {
 
     }
-
+    // TODO make version accurate - will return 3-0-0 even if version is 301/302
```

Comment:
We don't have the actual control block on hand here.  But this is only used by the unsupported() method.  And what we could do instead is update unsupported() to talk about version families or series (300 and 200) instead of specific versions.